geest/core/workflows/raster_reclassification_workflow.py

```diff
@@ -42,20 +42,20 @@ def __init__(
         )  # ⭐️ Item is a reference - whatever you change in this item will directly update the tree
         self.workflow_name = "use_environmental_hazards"
 
-        layer_name = self.attributes.get("Use Environmental Hazards Raster", None)
+        layer_name = self.attributes.get("use_environmental_hazards Raster", None)
```

Comment:
we should be using snake case for the complete attribute name so use_environmental_hazards_raster